.circleci/config.yml

```diff
@@ -305,9 +309,6 @@ jobs:
               && git push \
               || echo "Skipping push since it looks like there were no changes"
   deploy:
-    parameters:
-      image:
-        type: string
```

Comment:
This parameter was not used, Circle CLI validator started complaining about it after version bump.